packages/core-mobile/app/screens/send/hooks/usePVMSend.ts

```diff
@@ -70,19 +128,43 @@ const usePVMSend: SendAdapterPVM = ({
   )
 
   const validate = useCallback(async () => {
+    if (!provider) return
+    setError(undefined)
```

Comment:
setting error to undefined here was removed in this pr https://github.com/ava-labs/core-mobile/pull/2090/files#diff-e2f25834c471326b2bb3764bb171aaf64d1c58736a1eb7868a20aab236b6e18d cause it was not necessary to do so. is there a reason you added it back?

Reply:
it was added due to rebase

Reply:
will remove it.